README.md

```diff
@@ -4,14 +4,48 @@ These are the shared values of the betterplace.org engineering team. They exist
 
 ## We have solid engineering standards
 
+We are proud of the things we create, and use solid standards and best practices. At the moment those include:
+
+* Each pull request is reviewed by at least 2 developers before merging
```

Comment:
```suggestion
* Each pull request is reviewed by at least one other developer before merging
```
(i don't think 2 is realistic a.t.m. as there is often just one other dev who can judge the PR)